WordPress/src/androidTest/java/org/wordpress/android/e2e/EditorTests.java

```diff
@@ -0,0 +1,45 @@
+package org.wordpress.android.e2e;
+
+import android.support.test.rule.ActivityTestRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.wordpress.android.e2e.components.MasterbarComponent;
+import org.wordpress.android.e2e.components.SnackbarComponent;
+import org.wordpress.android.e2e.pages.EditorPage;
+import org.wordpress.android.e2e.pages.MySitesPage;
+import org.wordpress.android.support.BaseTest;
+import org.wordpress.android.ui.WPLaunchActivity;
+
+import static org.wordpress.android.BuildConfig.E2E_DEFAULT_USER_SITE_ADDRESS;
```

Comment:
Can you add all of these config properties to `gradle.properties-example`? That way anyone wanting to run the tests knows what properties to add there.

Reply:
Thanks for pointing this out, I've updated the example